src/modules/logger/logged_topics.cpp

```diff
@@ -204,7 +204,7 @@ void LoggedTopics::add_default_topics()
 	// add_optional_topic_multi("estimator_aid_src_optical_flow", 100, MAX_ESTIMATOR_INSTANCES);
 	// add_optional_topic_multi("estimator_aid_src_terrain_optical_flow", 100, MAX_ESTIMATOR_INSTANCES);
 	// add_optional_topic_multi("estimator_aid_src_ev_yaw", 100, MAX_ESTIMATOR_INSTANCES);
-	//add_optional_topic_multi("estimator_aid_src_aux_global_position", 100, MAX_ESTIMATOR_INSTANCES);
+	add_optional_topic_multi("estimator_aid_src_aux_global_position", 100, MAX_ESTIMATOR_INSTANCES);
```

Comment:
In keeping with the other aid source topics, should this be left commented out on `main`?

Reply:
This is the only aid source that is not part of the `estimator_innovation_xx` as it is in a separate class. That's why we absolutely need to log it. For the other sources, we'll also stop publishing the `estimator_innovation_xx` and use the `estimator_aid_src_` in the future as well.